go.mod

```diff
@@ -5,68 +5,69 @@ go 1.15
+	github.com/gardener/gardener v1.15.1-0.20210115062544-6dc08568692a
 	github.com/gardener/machine-controller-manager v0.35.0
-	github.com/go-logr/logr v0.1.0
+	github.com/go-logr/logr v0.3.0
 	github.com/gobuffalo/packr/v2 v2.8.1
 	github.com/golang/mock v1.4.4-0.20200731163441-8734ec565a4d
 	github.com/google/uuid v1.1.1
-	github.com/onsi/ginkgo v1.14.0
-	github.com/onsi/gomega v1.10.1
+	github.com/onsi/ginkgo v1.14.1
+	github.com/onsi/gomega v1.10.2
 	github.com/pkg/errors v0.9.1
 	github.com/sirupsen/logrus v1.6.0
-	github.com/spf13/cobra v0.0.6
+	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5
 	github.com/vmware/go-vmware-nsxt v0.0.0-20200114231430-33a5af043f2e
 	github.com/vmware/vsphere-automation-sdk-go/lib v0.3.1
 	github.com/vmware/vsphere-automation-sdk-go/runtime v0.3.1
 	github.com/vmware/vsphere-automation-sdk-go/services/nsxt v0.4.0
-	k8s.io/api v0.18.10
-	k8s.io/apiextensions-apiserver v0.18.10
-	k8s.io/apimachinery v0.18.10
-	k8s.io/apiserver v0.18.10
+	k8s.io/api v0.19.6
+	k8s.io/apiextensions-apiserver v0.19.6
+	k8s.io/apimachinery v0.19.6
+	k8s.io/apiserver v0.19.6
 	k8s.io/autoscaler v0.0.0-20190805135949-100e91ba756e
 	k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible
 	k8s.io/cloud-provider-vsphere v1.1.0
-	k8s.io/code-generator v0.18.10
-	k8s.io/component-base v0.18.10
-	k8s.io/kubelet v0.18.10
-	k8s.io/utils v0.0.0-20200619165400-6e3d28b6ed19
-	sigs.k8s.io/controller-runtime v0.6.3
+	k8s.io/code-generator v0.19.6
+	k8s.io/component-base v0.19.6
+	k8s.io/klog v1.0.0
+	k8s.io/kubelet v0.19.6
+	k8s.io/utils v0.0.0-20200912215256-4140de9c8800
+	sigs.k8s.io/controller-runtime v0.7.1
 	sigs.k8s.io/yaml v1.2.0
 )
 
 replace (
-	github.com/prometheus/client_golang => github.com/prometheus/client_golang v0.9.2
-	k8s.io/api => k8s.io/api v0.18.10
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.18.10
-	k8s.io/apimachinery => k8s.io/apimachinery v0.18.10
-	k8s.io/apiserver => k8s.io/apiserver v0.18.10
-	k8s.io/client-go => k8s.io/client-go v0.18.10
-	k8s.io/code-generator => k8s.io/code-generator v0.18.10
-	k8s.io/component-base => k8s.io/component-base v0.18.10
+	github.com/prometheus/client_golang => github.com/prometheus/client_golang v1.7.1 // keep this value in sync with k8s.io/client-go
+	k8s.io/api => k8s.io/api v0.19.6
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.19.6
+	k8s.io/apimachinery => k8s.io/apimachinery v0.19.6
+	k8s.io/apiserver => k8s.io/apiserver v0.19.6
+	k8s.io/client-go => k8s.io/client-go v0.19.6
+	k8s.io/code-generator => k8s.io/code-generator v0.19.6
+	k8s.io/component-base => k8s.io/component-base v0.19.6
 	k8s.io/helm => k8s.io/helm v2.13.1+incompatible
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.18.10
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.19.6
+	k8s.io/kube-openapi => github.com/gardener/kube-openapi v0.0.0-20201221124747-75e88872edcf // k8s-1.19
```

Comment:
Is this needed?

Reply:
Good question. I copied the line from the PR https://github.com/gardener/gardener/pull/3393 for consistency reasons.
see https://github.com/gardener/gardener/blob/master/go.mod#L85

This introduces this fix: https://github.com/gardener/kube-openapi/commit/75e88872edcfe2b3e9158cf29bce4d8f4459d1de ("fix handling of inlined embedded struct pointer fields")

Reply:
Yes, but do you use this in this repo? If not, I'd remove it.

Reply:
Removed as not used. Thanks for the hint.